app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailFragment.java

```diff
@@ -145,7 +130,6 @@ public void onRefreshComplete() {
 
     @Override
     public void showEvent(Event event) {
-        listener.onEventLoaded(event, false);
```

Comment:
yeah. it can.
but I am thinking, why the main presenter should load the event if it can use info from the detail fragment which will be loaded initially for sure. ??